### PR TITLE
Claim script

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Claim script

### CODE REVIEW

**Review by dvilelaf — APPROVED** [review]

**Review by dagacha — APPROVED** [review]
Tested - LGTM

File: claim_olas.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2023-2024 Valory AG
```
Comment (Adamantios) [inline claim_olas.sh:5]:
```suggestion
#   Copyright 2024 Valory AG
```

**Review by Adamantios — APPROVED** [review]
LGTM but lots of constants missing, e.g., safe's address, gas value, gas price, token precision, encoding, etc.

**Comment by jmoreira-valory** [comment]
> LGTM but lots of constants missing, e.g., safe's url, gas value, gas price, token precision, encoding, etc.

Addressed.

**Pull request merged**